### ISSUE TITLE
Garden

### ISSUE THREAD

**Issue by Sergel69** [Bugnon/minecraft#7]
Is my garden beautiful ?

**Comment by BenJonas** [comment]
yes your garden is really beautiful, you did a good innovation in minecraft world!!

**Comment by JoBenoit** [comment]
Yes it is,  I like the idea of a labyrinth veryy much. You can maybe do a true one. It would be great if you make a true one, where the goal would be to find an exit.

**Comment by MrLasagne4** [comment]
I would be nice to add flowers.

**Comment by Sergel69** [comment]
And what about water ?
